static/panes/compiler.js

```diff
@@ -1504,20 +1504,16 @@ Compiler.prototype.onLibsChanged = function () {
 };
 
 Compiler.prototype.initLibraries = function (state) {
-    var libUtils = new LibUtils();
```

Comment:
Does this play nice with older browesers?

Reply:
Maybe it's just better to refactor LibUtils out of the class, we should probably not have a class with only static methods, instead export standalone functions